app/uk/gov/hmrc/securemessage/services/Auditing.scala

```diff
@@ -119,15 +119,15 @@ trait Auditing {
     auditConnector.sendExplicitAudit(txnStatus, detail)
   }
 
-  def auditMessageForwarded(txnStatus: String, qrw: QueryResponseWrapper, eisResponseCode: Int)(
+  def auditMessageForwarded(txnStatus: String, qrw: QueryMessageRequest, eisResponseCode: Int)(
     implicit hc: HeaderCarrier,
     ec: ExecutionContext): Unit = {
     val detail = Map(
       txnName           -> "Message forwarded to caseworker",
       "eisResponseCode" -> eisResponseCode.toString,
-      "conversationId"  -> qrw.queryResponse.conversationId,
-      "x-request-id"    -> qrw.queryResponse.id,
-      "message"         -> qrw.queryResponse.message
+      "conversationId"  -> qrw.requestDetail.conversationId,
+      "x-request-id"    -> qrw.requestDetail.id,
```

Comment:
do we need a seetionId as well ?

Reply:
not sure, it's not the focus of my story